site-src/api-overview.md

```diff
@@ -33,6 +33,8 @@ Currently, there are two main objects in the Network Policy API resource model:
 
 - **BaselineAdminNetworkPolicy (BANP)**
 
+![Alt text](./images/ANP-api-model.png?raw=true "Admin Network Policy API Model")
```

Comment:
Can you ad a note here just something along the lines of 

"The below diagram demonstrates how these new API objects interact with each-other and existing NetworkPolicy Objects."

Reply:
done!